lib/ansible/modules/cloud/amazon/ec2_placement_group.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/python
+# Copyright (c) 2017 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: ec2_placement_group
+short_description: Create or delete an EC2 Placement Group
+description:
+    - Create an EC2 Placement Group; if the placement group already exists,
+      nothing is done. Or, delete an existing placement group. If the placement
+      group is absent, do nothing.
+version_added: "2.5"
+author: "Brad Macpherson (@iiibrad)"
+options:
+  name:
+    description:
+      - The name for the placement group.
+    required: true
+  state:
+    description:
+      - Create or delete placement group.
+    required: false
+    default: present
+    choices: [ 'present', 'absent' ]
+extends_documentation_fragment:
+    - aws
+    - ec2
+'''
+
+EXAMPLES = '''
+# Note: These examples do not set authentication details, see the AWS Guide
+# for details.
+
+# Create a placement group.
+- ec2_placement_group:
+    name: my-cluster
+    state: present
+
+# Delete a placement group.
+- ec2_placement_group:
+    name: my-cluster
+    state: absent
```

Comment:
Is there a reason you don't have a parameter to set the strategy? Seems like it'd be easy to add (and default to `cluster`).

Reply:
Simply that it isn't (or wasn't, until a day or so ago) an option as such. There was only one value.

Now that spread placement groups are available, that will have to change :-)

Reply:
Added parameter 'strategy'; attempting to provision a placement group with a different strategy to the existing one will fail.

Reply:
Hah - life with AWS: "oh, that's a parameter too now"